apps/opencs/CMakeLists.txt

```diff
@@ -225,6 +225,15 @@ if (WIN32)
     target_link_libraries(openmw-cs ${Boost_LOCALE_LIBRARY})
 endif()
 
+if (MSVC)
+    # Debug version needs increased number of sections beyond 2^16
+    if (CMAKE_CL_64)
+        set (CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} /bigobj")
+    endif (CMAKE_CL_64)
+    add_definitions("-D_USE_MATH_DEFINES")
```

Comment:
Is the addition of USE_MATH_DEFINES really necessary? Looks to me it was inadvertently copy-pasted.